lib/morse_translator.rb

```diff
@@ -1,43 +1,94 @@
-class MorseTranslator
+require 'pry'
 
+class MorseTranslator
+  attr_reader :translate
   def initialize
-    @dictionary = {"a" => ".-",
-                    "b" => "-...",
-                    "c" => "-.-.",
-                    "d" => "-..",
-                    "e" => ".",
-                    "f" => "..-.",
-                    "g" => "--.",
-                    "h" => "....",
-                    "i" => "..",
-                    "j" => ".---",
-                    "k" => "-.-",
-                    "l" => ".-..",
-                    "m" => "--",
-                    "n" => "-.",
-                    "o" => "---",
-                    "p" => ".--.",
-                    "q" => "--.-",
-                    "r" => ".-.",
-                    "s" => "...",
-                    "t" => "-",
-                    "u" => "..-",
-                    "v" => "...-",
-                    "w" => ".--",
-                    "x" => "-..-",
-                    "y" => "-.--",
-                    "z" => "--..",
-                    "1" => ".----",
-                    "2" => "..---",
-                    "3" => "...--",
-                    "4" => "....-",
-                    "5" => ".....",
-                    "6" => "-....",
-                    "7" => "--...",
-                    "8" => "---..",
-                    "9" => "----.",
-                    "0" => "-----",
-                    " " => " "}
+    @dictionary = {
+        "A" => ".-",
+        "B" => "-...",
+        "C" => "-.-.",
+        "D" => "-..",
+        "E" => ".",
+        "F" => "..-.",
+        "G" => "--.",
+        "H" => "....",
+        "I" => "..",
+        "J" => ".---",
+        "K" => "-.-",
+        "L" => ".-..",
+        "M" => "--",
+        "N" => "-.",
+        "O" => "---",
+        "P" => ".--.",
+        "Q" => "--.-",
+        "R" => ".-.",
+        "S" => "...",
+        "T" => "-",
+        "U" => "..-",
+        "V" => "...-",
+        "W" => ".--",
+        "X" => "-..-",
+        "Y" => "-.--",
+        "Z"  => "--..",
+        "a" => ".-",
```

Comment:
This is VERY repetitive!  Can you refactor this to work without duplicating the alphabet in the dictionary?